### PR TITLE
Increase timeout for waiting for kubectl proxy in test-cmd

### LINKED ISSUE #19252: integration flake: Timed out waiting for kubectl proxy
It seems like an issue similar to the problem with API server. I'll extend the timeout in the similar fashion.

cc @thockin @ixdy

### PR DESCRIPTION
**Pull request by gmarek**
Fixes #19252

### CODE REVIEW

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -55,9 +55,9 @@ function start-proxy()
 
   PROXY_PID=$!
   if [ $# -eq 0 ]; then
-    kube::util::wait_for_url "http://127.0.0.1:${PROXY_PORT}/healthz" "kubectl proxy"
+    kube::util::wait_for_url "http://127.0.0.1:${PROXY_PORT}/healthz" "kubectl proxy" 1 30
```
Comment (fgrzadkowski) [inline hack/test-cmd.sh]: Can you please add a comment that explains that you wait 1s 30 times?

Comment (gmarek) [inline hack/test-cmd.sh]: Done.

**Comment by gmarek** [comment]
ERROR: Error fetching remote repo 'origin'
@k8s-bot unit test this

**Comment by gmarek** [comment]
@k8s-bot unit test this

**Comment by mwielgus** [comment]
LGTM

**Comment by gmarek** [comment]
@k8s-bot unit test this

**Comment by gmarek** [comment]
ERROR: Error fetching remote repo 'origin'
@k8s-bot unit test this

**Pull request merged**